<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubMergeClass</class>
 <widget class="QMainWindow" name="SubMergeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SubMerge</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="B_Play">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PLAY</string>
    </property>
   </widget>
   <widget class="QLabel" name="Name">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>0</y>
      <width>38</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>10</horstretch>
      <verstretch>10</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Name :</string>
    </property>
   </widget>
   <widget class="QSlider" name="TimeLine">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>30</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="B_Pause">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>60</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PAUSE</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_trackName">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QSlider" name="s_Volume">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>80</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Volume</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>362</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFiles"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SubMerge.qrc"/>
 </resources>
 <connections/>
</ui>
